test/check-machines-disks

```diff
@@ -800,8 +797,8 @@ class TestMachinesDisks(VirtualMachinesCase):
         self.createVm("subVmTest1")
 
         # Prepare file for Custom File disk type
-        m.execute("qemu-img create -f raw /var/lib/libvirt/novell.iso 1M")
-        m.execute("qemu-img create -f qcow2 /var/lib/libvirt/novell.qcow2 1M")
```

Comment:
Was that by accident, or on purpose? I do see that using qemu-img for a fake iso file is wrong, but for a .qcow2 file it still seems right?

Reply:
I just reverted the commit completely, with git revert. Yes. this should be fine.